src/index.html

```diff
@@ -1,13 +1,32 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img src="images/imac.jpeg" alt="Моноблок APPLE A1419 iMac 27' Retina 5K (MNED2UAA) class='card-picture'">
```

Comment:
```suggestion
      <img 
          src="images/imac.jpeg"
          alt="Моноблок APPLE A1419 iMac 27' Retina 5K (MNED2UAA)" 
          class="card-picture'"
      >
```
Коли у тега більше ніж 3 атрибута, то переносимо кожен на новий рядок